test/WebJobs.Script.Tests.Integration/Diagnostics/MetricsEventManagerTests.cs

```diff
@@ -66,7 +67,8 @@ public MetricsEventManagerTests()
                         Average = average,
                         Minimum = min,
                         Maximum = max,
-                        Count = count
+                        Count = count,
```

Comment:
I think order of parameters is important. Count needs to be the last one.

Reply:
yes, I tried private site extension and it works.
It looks like the order doesn't matter:
https://github.com/Azure/azure-functions-host/blob/700aa23ba8185e408d78fac00a722e0d010dd02a/src/WebJobs.Script.WebHost/Diagnostics/ExtendedEventSource.cs#L209